src/server.rs

```diff
@@ -278,8 +279,12 @@ messages! {
         "workspace/didChangeConfiguration" => WorkspaceChangeConfiguration(DidChangeConfigurationParams);
     }
     // TODO handle me
-    "$/setTraceNotification" => Err(ParseError::new(ErrorKind::InvalidData, "setTraceNotification", None));
-    _ => Err(ParseError::new(ErrorKind::InvalidData, "Unknown command", None));
+    "$/setTraceNotification" => Err(jsonrpc::Error {
+                                       code: jsonrpc::ErrorCode::MethodNotFound,
+                                       message: String::from("setTraceNotification"),
+                                       data: None
+                                    });
```

Comment:
Not really, just tried to retain the same semantics when rewriting it. Should I move it under _?